config/src/main/java/io/confluent/common/config/ConfigDef.java

```diff
@@ -336,6 +336,27 @@ else if (value instanceof String)
           } else {
             throw new ConfigException(name, value, "Expected a comma separated list.");
           }
+        case MAP:
+          if (value instanceof Map) {
+            return value;
+          } else if (value instanceof String) {
+            if(trimmed.isEmpty()) {
+              return Collections.emptyMap();
+            } else {
+              Map<String, String> map = new HashMap<>();
+              List<String> entries = Arrays.asList(trimmed.split("\\s*,\\s*", -1));
+              for ( String entry : entries) {
+                String[] keyValue = entry.split("\\s*:\\s*", -1);
```

Comment:
same comment as in the other PR: "=" as delimiter seems more natural. (but I don't really have anything to back that up)

Reply:
I based it off the config in Kafka listener.security.protocol.map. The separator here is ':'